src/main/java/Duke.java

```diff
@@ -6,5 +8,76 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        String line;
+        String list = "list";
+        String bleh = "bleh";
+        String bye = "bye";
+        Scanner in = new Scanner(System.in);
+        Task[] taskList = new Task[100];
+        int i = 0;
+
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Duke");
+        System.out.println(" What can I do for you?");
+        do {
+            line = in.nextLine();
+
+            if (line.equals(bleh)) {
+                System.out.println("____________________________________________________________");
+                System.out.println(" bleh");
+                System.out.println("____________________________________________________________");
+            } else if(line.equals(list)){
+                System.out.println("____________________________________________________________");
+                int number = 1;
+                System.out.println(" Here are the tasks in your list:");
+                while (number <= i) {
+                    System.out.println(number + ":[" + taskList[number-1].getStatusIcon()+ "] " + taskList[number-1].description);
+                    number++;
+                }
+                System.out.println("____________________________________________________________");
+            } else if(line.contains("done")){
+                int dividerPosition = line.indexOf(" ");
+                String taskNumber = line.substring(dividerPosition+1);
+                int number = Integer.parseInt(taskNumber);
```

Comment:
Perhaps this variable could be named more clearly so it's more obvious what it's for?